hover.com/extract.js

```diff
@@ -0,0 +1,29 @@
+// CONFIG BEGIN
+var defaultTTL = 14400;
+// CONFIG END
+
+var table = document.getElementsByClassName('table-body')[0];
+var rows = table.getElementsByClassName('table-row');
+var i, len, row;
+var hostname, type, priority, ttl, destination;
+var output = '';
+
+output += '$TTL ' + defaultTTL + '\n'; // start with default TTL
+
+// skip header and last two rows (add new entry, delete all entries)
+for (i = 0, len = rows.length; i < len; i++) {
+  row = rows[i];
+  hostname = row.getElementsByClassName('name')[0].innerText;
+  type = row.getElementsByClassName('type')[0].innerText;
+  // priority = row.getElementsByClassName('dns_priority')[0].innerText; - not included
```

Comment:
Feel free to remove this line - I see from screen shots on [Managing DNS Records](https://help.hover.com/hc/en-us/articles/217282457-Managing-DNS-records-) that the priority is included in the value (`content`) column.